Makefile

```diff
@@ -56,10 +54,6 @@ check_docker:
 check_base_files:
```

Comment:
the docker linter [target](https://github.com/terraform-google-modules/terraform-google-cloud-datastore/pull/12/files#diff-b67911656ef5d18c4ae36cb6741b7965R49-R51) is deprecated and to be removed